### PR TITLE
Read ciflow labels from config

### PR DESCRIPTION
**Pull request by malfet**
I.e. don't hardcode them

Fixes https://github.com/pytorch/test-infra/issues/1439

### CODE REVIEW

**Comment by seemethere** [comment]
Can we get an example here of how a configuration is supposed to look like?

File: torchci/lib/bot/ciflowPushTrigger.ts [hunk 1]
```diff
@@ -133,54 +136,24 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
...
-    "ciflow/binaries_wheel",
-    "ciflow/inductor-perf-test-nightly",
-  ];
+  const config: any = await tracker.loadConfig(context);
+  const valid_labels: Array<string> =
+    config !== null ? config["ciflow_push_tags"] : null;
+  if (valid_labels == null) {
+    await context.octokit.issues.createComment(
+      context.repo({
+        body: "No ciflow labels are configured for this repo.",
```
Comment (seemethere) [inline torchci/lib/bot/ciflowPushTrigger.ts:145]: A link to some documentation on how one can create a ciflow config would be a good quality of life improvement here

**Review by seemethere — APPROVED** [review]
Looks mostly good to me, just worried about user experience for people trying to onboard to this.

**Review by huydhn — APPROVED** [review]
LGTM!

**Comment by huydhn** [comment]
> Can we get an example here of how a configuration is supposed to look like?

Here is what it currently looks like for PyTorch https://github.com/pytorch/pytorch/blob/master/.github/pytorch-probot.yml#L3-L14 I think

File: torchci/lib/bot/ciflowPushTrigger.ts (hunk 1 above)
Comment (malfet) [reply torchci/lib/bot/ciflowPushTrigger.ts]: https://github.com/pytorch/test-infra/wiki/PyTorch-bot#ciflow-bot

**Pull request merged**